djangogirls/mysite/__init__.py

```diff
@@ -0,0 +1 @@
+"""Django Project."""
```

Comment:
This is too generic. It doesn't give any hint about what the project does. Also, it's not a project but a package that you describe. Although, it doesn't even matter because this info is useless. You could write that it's a blog or something like that instead

Reply:
ok